docs/painless/painless-contexts/painless-ingest-processor-context.asciidoc

```diff
@@ -38,4 +38,145 @@ void::
 
 *API*
 
-The standard <<painless-api-reference, Painless API>> is available.
\ No newline at end of file
+The standard <<painless-api-reference, Painless API>> is available.
+
+*Example*
+
+To use this example first follow the steps outlined in
+<<painless-context-examples, context examples>>.
+
+The original seat data gives a date `String` with the format `YYYY-MM-DD` where
+the second digit of each the month and the day is optional and gives a time
+`String` with the format `HH:MM*` where the second digit of each the hours and
+the minutes is optional and the star represents a `String` of either `AM` or
+`PM`. The following ingest script processor converts the given date and time
+`Strings` into a useful format for storage into a date field. Upon completion of
+the script each document will have its `datetime` field value filled in.
+
```

Comment:
Strictly editorial...I'd say:

The seat data contains:

* A date in the format `YYYY-MM-DD` where the second digit of both month and day is optional.
* A time in the format `HH:MM*` where  the second digit of both hours and minutes is optional. The star (*) represents either the String `AM` or `PM`. 

The following ingest script processes the date and time Strings and stores the result in a `datetime`field.

Reply:
That's much better.  Fixed.